src/documentation/pages/components/icon/examples/_brand.vue

```diff
@@ -0,0 +1,74 @@
+<template lang="pug">
+<ComponentExample title="Brand" titleSize="h4" id="brand" padding="-p--0" :tabs="exampleTabs">
+  p.-text(slot="example-description")
+    | Brand icon colors reinforce Lumen's brand.
+  .chi-grid.-no-gutter(slot="example")
```

Comment:
Can we use this?

```suggestion
  <Wrapper slot="example">
```

Within the Wrapper we may be able to use `.chi-grid.-no-gutter` and other classes.